src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write code here
+const inputs = document.querySelectorAll('input');
+const upperChart = (str) => {
+  return str[0].toUpperCase() + str.slice(1);
+};
+
+[...inputs].map((item) => {
```

Comment:
```suggestion
[...inputs].forEach((item) => {
```
map тебе тут не нужен